db/compilers.xml

```diff
@@ -24,7 +24,7 @@
     <version>
       <external>$EXEC -v</external>
       <nogrep regexp="for GNAT Pro"></nogrep>
-      <grep regexp="^gcc \S+ (\S+)" group="1"></grep>
+      <grep regexp="^gcc[- ]\S+ (\S+)" group="1"></grep>
```

Comment:
Maybe using 

```suggestion
      <grep regexp="^gcc.\S+ (\S+)" group="1"></grep>
```

is better. In that case any character betwen `gcc` and `[V,v]ersion` can be matched.